### PR TITLE
Completed the Jupyter Notebook

### LINKED ISSUE #2: Jupyter Notebook Setup
As the previous engineer left in a hurry, the model that was provided to you is pre-trained and you do not have any information on how it was trained. You are tasked with evaluating the model's performance and fairness.

When it comes to training the model itself, the ML engineer who worked on the project before you had the following assumptions:

The model is used to predict whether a student will be a good candidate for a software engineering job. It is a binary classifier, where 1 means the student is a good candidate, and 0 means the student is not a good candidate.
The model is trained on a dataset of students who have graduated from CMU, and have been working in the industry for at least 1 year.
In order to prevent bias, they assumed that removing Gender (M, F) and Student ID from the dataset would be sufficient when it comes to training the model. They claimed that it is now Group Unaware, thus the model would be fair.

You are provided with a [test dataset](https://cmu-17313q.github.io/assets/project/student_data.csv), which contains a similar set of features and output (whether the student is a good candidate or not). This test dataset is a different set of students from the training dataset, and the evaluation of whether the student is a good candidate is done by a fair panel of recruiters, so it can be considered to be unbiased. Additionally, the panel of recruiters have provided you with additional context on the extracurricular activities in comments (marked with #).

Before doing a thorough evaluation of the fairness of the model, you will start by doing preliminary analysis on the [test dataset](https://cmu-17313q.github.io/assets/project/student_data.csv), and run the model on the test dataset to get the accuracy of the model. To do so, you will need to set up a Jupyter notebook to do this, you can either:

After you have set up the notebook, you should:

1. Load the model and test dataset
2. Plot the distribution of the test dataset across all features (except Student ID) using any visualization library of your choice (e.g. 
pandas, matplotlib, seaborn, plotly, etc.). You should choose the appropriate visualization for each feature.
3. Predict the output of the test dataset using the model
4. Report the accuracy of the model, and the confusion matrix

### PR DESCRIPTION
**Pull request by hjoad**
Resolves #2 
1. Loaded the model and test dataset
2. Plotted the distrubtion of the test dataset across all features (except Student ID) using seaborn.
3. Predicted the output of the test dataset using the model.
4. Reported the accuracy of the model and the confusion matrix.

### CODE REVIEW

**Review by Maria-Aidarus — APPROVED** [review]
All clear Alhamdulilah :)
